frontend/src/lib/components/Users/UserFilter.svelte

```diff
@@ -52,7 +59,19 @@
 >
   <svelte:fragment slot="activeFilters" let:activeFilters>
     {#each activeFilters as filter}
-      {#if filter.key === 'usersICreated' && filter.value}
+      {#if filter.key === 'userType' && filter.value}
+        <ActiveFilter {filter}>
+          {#if filter.value === 'admin'}
+            <Icon icon="i-mdi-shield-lock-outline" color="text-warning" />
```

Comment:
We already have an icon for admins, using the `i-mdi-security` icon (which displays as a shield). Let's keep it consistent by using `i-mdi-security` here as well. It also uses the `text-accent` color, which again, should probably be consistent here as well.